Solutions/Microsoft Defender Threat Intelligence/Playbooks/readme.md

```diff
@@ -0,0 +1,43 @@
+# MDTI Playbook Guide
+
+![Microsoft Defender Threat Intelligence](./MDTI.jpg)<br>
+
+## Table of Contents
+
+1. [Overview](#overview)
+1. [Authentication](#authentication)
+1. [Deployment](#deployment)
+1. [Post Deployment Steps](#postdeployment)
+
+<a name="overview">
+
+# Overview
+Microsoft centralizes numerous data sets into a single platform, Microsoft Defender Threat Intelligence [(MDTI)](https://learn.microsoft.com/en-us/defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti), making it easier for Microsoft’s community and customers to conduct infrastructure analysis. Microsoft’s primary focus is to provide as much data as possible about Internet infrastructure to support a variety of security use cases. If you have trouble accessing your account or your credentials contact your account representative or reach out to discussMDTI[@]microsoft.com
+
+# MDTI Playbook Documentation
+
+<a name="authentication">
+
+## Authentication
+Azure AD App Registration credentials(ClientId/ClientSecret/TenantId) with MDTI API Permissions are needed when configuring this playbook. Those can be found on your [Azure Client App](https://learn.microsoft.com/en-us/rest/api/azure/#register-your-client-application-with-azure-ad) page.
+
+MDTI API documentation for more details, [MDTI API](https://learn.microsoft.com/en-us/graph/api/resources/security-threatintelligence-overview?view=graph-rest-beta&branch=pr-en-us-20472).
```

Comment:
The Link for MDTI API Documentation is correct, it will go live on March 28th. you can validate the link by adding review in front of it. like this, (https://review.learn.microsoft.com/en-us/graph/api/resources/security-threatintelligence-overview?view=graph-rest-beta&branch=pr-en-us-20472)